packages/clay-layout/src/Row.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+type TNumberOfColumns = 1 | 2 | 3 | 4 | 5 | 6;
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Vertical positioning of row contents
+	 */
+	align?: 'start' | 'center' | 'end';
+
+	/**
+	 * Element or component to render for container
+	 */
+	containerElement?: React.JSXElementConstructor<{
+		className: string;
+		[key: string]: any;
+	}>;
+
+	/**
+	 * This removes the negative margins from .row and the
+	 * horizontal padding from all immediate children columns
+	 */
+	gutters?: boolean;
+
+	/**
+	 * Horizontal positioning of row contents
+	 */
+	justify?: 'start' | 'center' | 'end' | 'around' | 'between';
```

Comment:
Same as `align-items`, has more options than the ones listed here.

Reply:
Same as above comment, I am trying to keep it minimal and follow bootstraps grid docs for what should be available.